src/cloudant/document.py

```diff
@@ -158,7 +158,7 @@ def create(self):
         super(Document, self).__setitem__('_rev', data['rev'])
         return
 
-    def fetch(self, rev=None):
+    def fetch_revision(self, rev):
```

Comment:
The doc comment says that the default for `rev` is `None`.  Is there a reason why you removed `rev=None` in the function signature?